libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -494,6 +495,7 @@ private void initializeView() {
   private void bind() {
     mapView = findViewById(R.id.navigationMapView);
     instructionView = findViewById(R.id.instructionView);
+    ViewCompat.setElevation(instructionView, 10);
```

Comment:
Have we considered extracting the magic number `10` into a constant specifying that this is actually putting the `instructionView` at the top of the view hierarchy?